qubes-rpc/post-install.d/95-fstrim.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/sh
+
+# perform fstrim on all partitions after finishing update-related I/O
+
+# abort if not in a template
+if [ "$(qubesdb-read /type)" = "TemplateVM" ]
```

Comment:
Maybe better check `/qubes-vm-persistence` to be `full`, or simply check for `/run/qubes/persistent-full` existence? It would then cover standalone qubes too.